specification/loadtestservice/LoadTestService.Management/models.tsp

```diff
@@ -0,0 +1,258 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-providerhub";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.ResourceManager;
+using OpenAPI;
+using Autorest;
+using Azure.ResourceManager.Foundations;
+using Azure.ResourceManager.Private;
+
+namespace Microsoft.LoadTestService;
+
+@doc("LoadTest details.")
+model LoadTestResource is TrackedResource<LoadTestProperties> {
+  @doc("Load Test name")
+  @key("loadTestName")
+  @extension("x-ms-parameter-location", "method")
```

Comment:
Don't think this is needed.

Reply:
We are removing it, but the "x-ms-parameter-location": "method", is already defined in GA spec, let us know if it causes any issue in SDK if we remove this now.

Reply:
I think by default is method. Hence the ask to remove.